device-telemetry/app/com/microsoft/azure/iotsolutions/devicetelemetry/services/StatusService.java

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+package com.microsoft.azure.iotsolutions.devicetelemetry.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.Inject;
+import com.microsoft.azure.iotsolutions.devicetelemetry.services.helpers.WsRequestBuilder;
+import com.microsoft.azure.iotsolutions.devicetelemetry.services.models.StatusResultServiceModel;
+import com.microsoft.azure.iotsolutions.devicetelemetry.services.models.StatusServiceModel;
+import com.microsoft.azure.iotsolutions.devicetelemetry.services.runtime.IServicesConfig;
+import com.microsoft.azure.iotsolutions.devicetelemetry.services.runtime.StorageType;
+import com.microsoft.azure.iotsolutions.devicetelemetry.services.storage.cosmosDb.IStorageClient;
+import com.microsoft.azure.iotsolutions.devicetelemetry.services.storage.timeSeries.ITimeSeriesClient;
+import org.apache.http.HttpStatus;
+import play.Logger;
+import play.libs.ws.WSResponse;
+
+import java.util.ArrayList;
+
+public class StatusService implements IStatusService {
+
+    private final String storageAdapterName = "StorageAdapter";
+    private final String diagnosticsName = "Diagnostics";
+    private final String authName = "Auth";
+
+    private final IStorageClient storageClient;
+    private final ITimeSeriesClient timeSeriesClient;
+    private final WsRequestBuilder wsRequestBuilder;
+    private final IServicesConfig servicesConfig;
+
+    private static final Logger.ALogger log = Logger.of(StatusService.class);
+
+    @Inject
+    StatusService(
+        IStorageClient storageClient,
+        ITimeSeriesClient timeSeriesClient,
+        IServicesConfig servicesConfig,
+        WsRequestBuilder wsRequestBuilder) {
+        this.storageClient = storageClient;
```

Comment:
nit: keep the order the same as the parameters.

Reply:
done